src/components/Footer/SocialLinks/SocialLinks.test.tsx

```diff
@@ -40,7 +40,9 @@ describe('SocialLinks component', () => {
 describe('SocialLink component', () => {
   it('renders without errors', () => {
     const { container } = render(<SocialLink name="Instagram" />)
-    expect(container.querySelector('.usa-social-link')).toBeInTheDocument()
-    expect(screen.getByRole('img')).toHaveAttribute('alt', 'Instagram')
```

Comment:
Question: Should these have alt tags too?

Reply:
no, the `alt` prop only applies to `<img>`. Now that it's an `<svg>`, I have a line at the bottom of that block to check the `name` prop instead, which serves the same purpose.